lib/web/apiserver.go

```diff
@@ -1084,44 +1084,7 @@ type nodeWithSessions struct {
 	Sessions []session.Session `json:"sessions"`
 }
 
-type getSiteNodesResponse struct {
-	Nodes []nodeWithSessions `json:"nodes"`
-}
-
-/* getSiteNodes returns a list of nodes active in the site
-
-GET /v1/webapi/namespaces/:namespace/sites/:site/nodes
-
-Sucessful response:
-
-{"nodes": [
-  {
-    "node": {
-        "addr": "ip:port",
-        "hostname": "a.example.com",
-        "labels": {"role": "mysql"}, // static key value pairs set by user for every node
-        "cmd_labels": {
-            "db_status": {
-               "command": "mysql -c status", // command periodically executed on server
-               "result": "master",  // output of the command
-               "period": 1000000000 // microseconds between calls
-             }
-        }
-     },
-     "sessions": [{
-         "id": "unique session id",
-         "parties": [{ // parties is a list of currently active participants
-            "id": "party id",
-            "user": "alice", // teleport user
-            "server_addr": "127.0.0.1:3000",
-            "last_active": "time" // RFC3339 timestamp when user was last acive
-         }]
-     }]
-   }
-  ]
-}
-*/
-func (m *Handler) getSiteNodes(w http.ResponseWriter, r *http.Request, p httprouter.Params, c *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+func (m *Handler) siteNodesGet(w http.ResponseWriter, r *http.Request, p httprouter.Params, c *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
```

Comment:
Do we not use this endpoint to get sessions anymore?

Reply:
Correct. We stopped using it a while back and did not clean the webapi endpoint.